app/pb_kits/playbook/pb_nav/_nav.jsx

```diff
@@ -4,6 +4,9 @@ import NavItem from "./_item"
 import {Caption} from "../"
 
 const propTypes = {
```

Comment:
The reason this one uses `PropTypes` is because it was created long ago before we instituted FlowJS (mentioned above). I do feel like if we are changing enough props here then, at some point, the need is to convert over to FlowJS types.